ui/StatusQ/src/StatusQ/Core/Utils/StackViewStates.qml

```diff
@@ -0,0 +1,45 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+
+QtObject {
+    id: root
+
+    required property StackView stackView
+
+    readonly property StatesStack _statesStack: StatesStack { id: statesStack }
+
+    property alias currentState: statesStack.currentState
+    property alias size: statesStack.size
+    property alias states: statesStack.states
+
+    function pushInitialState(state) {
+        if(size > 0)
+            console.warn("Pushing initial state but the stack already contains elements:  " + size)
+        _statesStack.push(state)
```

Comment:
`stateStack` is accessed via both in and `_statesStack` prop name. What about switching to `id` only?

Reply:
Sure!